Resources/Prototypes/_White/Loadouts/Jobs/medical.yml

```
@@ -0,0 +1,96 @@
+
+#Chemist
+- type: loadout
+  id: LoadoutClothingUniformJumpsuitMedicalUniformChemistry
+  category: JobsMedicalChemist
+  cost: 0
+  exclusive: true
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutChemistUniforms
+    - !type:CharacterJobRequirement
+       jobs:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)